lib/ui/widgets/link_web_view.dart

```diff
@@ -0,0 +1,27 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+
+// MODELS
+import 'package:notredame/core/models/quick_link.dart';
+
+// OTHER
+import 'package:webview_flutter/webview_flutter.dart';
+
+class LinkWebView extends StatelessWidget {
+  final QuickLink _links;
+
+  const LinkWebView(this._links);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(_links.name),
+      ),
+      body: WebView(
```

Comment:
Is it possible to add something saying the page is loading? At least the user will know that something is happening for a slow loading website like News

Reply:
Parfait ! Je fais ça d'ici le souper :D